go_api/domain/admin.go

```diff
@@ -0,0 +1,14 @@
+package domain
+
+import (
+	"time"
+)
+
+type Admin struct {
+	ID        int       `json:"id" gorm:"primary_key;not null"`
```

Comment:
```suggestion
	ID        uint       `json:"id" gorm:"primary_key;not null"`
```
ここは`uint`のほうが適切かもしれないですね